client/pages/my-garage.jsx

```diff
@@ -96,23 +100,25 @@ class MyCars extends React.Component {
       });
       return;
     }
-    const carData = { year, make, model };
+    const formData = new FormData();
+    formData.append('year', year);
+    formData.append('make', make);
+    formData.append('model', model);
+    formData.append('photoUrl', this.fileInputRef.current.files[0]);
     fetch('/api/garage/add-car', {
       method: 'POST',
-      headers: {
-        'Content-Type': 'application/json'
-      },
-      body: JSON.stringify(carData)
+      body: formData
     })
       .then(res => res.json())
       .then(data => {
-        this.getCars();
         this.setState({
+          cars: this.state.cars.concat([data]),
           year: '',
           make: '',
           model: '',
           modal: !this.state.modal
         });
```

Comment:
I didn't catch this before, but if your intent is to close the modal, just set it to `false` explicitly.

```js
modal: false
```